lux/test/unit/lux/discord_lens_test.exs

```diff
@@ -0,0 +1,108 @@
+defmodule Lux.Lenses.DiscordLensTest do
+  use ExUnit.Case
+  import Mock
+
+  alias Lux.Lenses.DiscordLens
+
+  setup do
+    with_mock Lux.Config, [
+      discord_bot_token: fn -> "test_bot_token" end
+    ] do
+      :ok
+    end
+  end
+
+  describe "focus/1" do
+    test "successfully makes GET request" do
+      response = %{"data" => "test"}
+
+      with_mocks([
+        {Req, [], [
+          new: fn _opts -> {:ok, %{}} end,
+          request: fn _req, _opts ->
+            {:ok, %{status: 200, body: response}}
+          end
+        ]},
+        {Lux.Config, [], [
```

Comment:
Please check other Lenses tests where we simply use `Req.Test`. Let's not try to mock `Lux.Config`.